queries.sql

```diff
@@ -133,4 +133,68 @@ GROUP BY owners.full_name
 ORDER BY animal_count DESC
 LIMIT 1;
 
+/* Queries for visits, specializations and vets */ 
+SELECT animals.name
+FROM animals
+JOIN visits ON animals.id = visits.animal_id
+JOIN vets ON visits.vet_id = vets.id
+WHERE vets.name = 'Vet William Tatcher'
+ORDER BY visits.visit_date DESC
+LIMIT 1;
+
+SELECT COUNT(DISTINCT visits.animal_id)
+FROM visits
+JOIN vets ON visits.vet_id = vets.id
+WHERE vets.name = 'Vet Stephanie Mendez';
```

Comment:
- [ ] Stephanie Mendez visited 4 animals but your query resulted in `3`, kindly consider fixing it.